app/controllers/follows_controller.rb

```diff
@@ -85,6 +85,12 @@ def update
     redirect_to "/dashboard/following" if @follow.update(follow_params)
   end
 
+  def destroy
+    @follow = Follow.find(params[:id])
+    authorize @follow
+    redirect_to "/dashboard/following" if @follow.destroy
```

Comment:
what happens if the destroy doesn't work?

ps. you can replace the hardcoded path with `dashboard_following_path`